torch/_inductor/codegen/cuda/cuda_cpp_scheduling.py

```diff
@@ -66,7 +66,9 @@ def define_kernel(self, src_code: str, node_schedule) -> str:
             compile_wrapper = IndentedBuffer()
             compile_wrapper.writeline("async_compile.cuda(r'''")
             compile_wrapper.splice(src_code, strip=True)
-            compile_wrapper.writeline("''', 'so')")
+            compile_wrapper.writeline(
+                f"''', 'so', aot_compile={str(V.graph.aot_mode)})"
+            )
```

Comment:
When we compile kernels in the JIT step, then we also pass a flag to tell our CUDACodeCache to compile an object file for us. So, we can eventually link it.

I have a feeling this isn't the best way to do it, but this approach probably involves the least code changes.